docker/php/conf.d/symfony.dev.ini

```diff
@@ -1,11 +1,14 @@
 apc.enable_cli = 1
 date.timezone = UTC
-session.auto_start = Off
-short_open_tag = Off
 
 # http://symfony.com/doc/current/performance.html
 opcache.interned_strings_buffer = 16
 opcache.max_accelerated_files = 20000
 opcache.memory_consumption = 256
+enable_file_override = 1
```

Comment:
According to Blackfire, it's better to enable this and to disable timestamp validation.